cli/tests/e2e/snapshots/test_semgrep_core_parse_error/test_file_parser__failure__error_messages/settings1/error.txt

```diff
@@ -45,7 +45,7 @@ Files skipped:
 
   [1m[24mPartially analyzed due to parsing or internal Semgrep errors[0m
 
-   • [36m[22m[24mtargets/bad/invalid_python.py (2 lines skipped)[0m
+   • [36m[22m[24mtargets/bad/invalid_python.py (3 lines skipped)[0m
```

Comment:
Do you know why your PR has an impact on this snapshot?

Reply:
I don't, my assumption was that because I updated how the AST is reported when printed out (by adding an extra field) it added an extra line somewhere in an error message that was printing out some of the AST. I will manually run these tests to confirm that is the case.

Reply:
It doesn't as it turns out. This is broken on my machine for some reason, even on develop.